files/common/lib/systemd/system/delphix-platform.service

```diff
@@ -19,6 +19,14 @@ Description=Delphix Appliance Platform Service
 After=local-fs.target
 Before=rsync.service
 
+#
+# This file is created before performing an OS migration. It is cleared
+# after a succesful run of the delphix-migration service. If the latter
+# has failed then it is unsafe for this service nor its dependencies to
+# run.
+#
+ConditionPathExists=!/var/delphix/migration/perform-migration
```

Comment:
Can you comment on under what circumstances this condition would trigger, and prevent this service from running?

Since this service requires `delphix-migration`, I don't understand the purpose of this condition.

Reply:
We talked about this offline, so I'm just putting this for the comments. This is just an additional failsafe that kicks in if delphix-migration is manually disabled before it has run successfully. Also, it should be replaced by `ExecStartPre`.